static/js/lib/coupon.js

```diff
@@ -25,28 +25,26 @@ function makeProgramIdLookup<T: HasProgramId>(obj: Array<T>): Map<number, T> {
   return new Map(obj.map((value: T) => [value.program_id, value]));
```

Comment:
can we rename the `obj` argument to be `xs` or `arr` or similar? `obj` isn't so appropriate a name for an array imo.